src/legacy/core_plugins/table_vis/index.js

```diff
@@ -27,7 +28,12 @@ export default function (kibana) {
         'plugins/table_vis/table_vis'
       ],
       styleSheetPaths: resolve(__dirname, 'public/index.scss'),
-    }
+      migrations: {
```

Comment:
I think you could just make this `migrations,` and get rid of the `{ visualization: { '7.0.0': migrations.visualization['7.0.0'] } }` bit. If someone wants to add non-visualization migrations to this in the future, they could just use the spread operator then.